core/src/main/web/app/mainapp/components/notebook/newcellmenu.jst.html

```diff
@@ -17,9 +17,16 @@
   <button ng-click="newCodeCell(defaultEvaluator())" class="btn btn-primary insert-cell" ng-class="size && 'btn-'+size">
     Insert {{defaultEvaluator()}} Cell
   </button>
+  <bk-language-logo ng-if="!last"
+                    class="advanced-only"
+                    ng-click="newCodeCell(defaultEvaluator())"
+                    bg-color="{{getEvaluatorDetails(defaultEvaluator()).bgColor}}"
```

Comment:
Seems suspicious to need {{ }} here since angular directive attributes are evaluated on the scope during the compile/link phase right?